project1/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>The Crane Wife: Short Story by CJ Hauser_Project 1</title>
+    <link rel="stylesheet" href="styles.css">
+    <link
+        href="https://fonts.googleapis.com/css?family=Aleo:400,400i,700i|Courgette|Denk+One|Fauna+One|Glegoo:400,700|Montserrat:300,300i,400,600,600i|Petit+Formal+Script|Satisfy|Syncopate:400,700&display=swap&subset=latin-ext"
```

Comment:
You're loading a lot more fonts and font weights here than are actually in use in the project. While this is fine for while you're working on the project, before you turn it in and consider it "final," you should make sure you're only calling for the fonts you use/need:
```suggestion
        href="https://fonts.googleapis.com/css?family=Denk+One|Montserrat:400|Satisfy&display=swap&subset=latin-ext"
```